src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1570,9 +1570,9 @@ internal enum ErrorCode
 
         ERR_UnmanagedBoundWithClass = 8380,
 
-        ERR_TupleSizesMismatchForBinOps = 8373,
-        ERR_ExpressionTreeContainsTupleBinOp = 8374,
-        WRN_TupleBinopLiteralNameMismatch = 8375,
```

Comment:
When I merged latest dev15.7.x bits into features/tuple-equality branch yesterday, I had to do a force push because github kept giving me unicorns. This is fixing a merge issue.